.github/include/ci.py

```diff
@@ -274,6 +276,34 @@ def test():
             ),
         )
     )
+    results.append(
+        test_one(
+            "runtime-tests.sh (AOT)",
+            lambda: truthy(RUN_AOT_TESTS),
+            lambda: shell(
+                (
+                    [
+                        "./tests/runtime-tests.sh",
+                        "--run-aot-tests",
+                        "--filter",
+                        "aot.*",
+                    ]
+                    + [
+                        "--skiplist_file",
+                        f"{root()}/{AOT_SKIPLIST_FILE}",
+                    ]
+                    if AOT_SKIPLIST_FILE
```

Comment:
nit: why do we need this check here? Are there cases when it's ok if CI doesn't find this file even though we specify it.